src/Playwright/Transport/PlaywrightLogger.cs

```diff
@@ -0,0 +1,68 @@
+using System;
+using System.Collections.Concurrent;
+using System.Collections.Generic;
+using System.Text.RegularExpressions;
+using Microsoft.Extensions.Logging;
+
+namespace Microsoft.Playwright.Transport
+{
+    internal class PlaywrightLogger : ILogger
+    {
+        private static readonly Dictionary<string, ConsoleColor> _debugLoggerColorMap = new()
+        {
+            { "pw:api", ConsoleColor.Cyan },
```

Comment:
can we drop that color map? I think not needed for .NET. and this would simplify it and reduce the maintenance overhead.